service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/digitaltwin/generated/DigitalTwins.java

```diff
@@ -14,7 +14,7 @@
 import com.microsoft.rest.ServiceCallback;
 import com.microsoft.rest.ServiceFuture;
 import com.microsoft.rest.ServiceResponseWithHeaders;
-import java.io.IOException;
+
```

Comment:
Nit: Did cleanup added a new line here?

Reply:
Yeah I believe so, I didn't touch any of the files after running the tool.